### PR TITLE
Add form to add, edit or delete a clinic

### LINKED ISSUE #10: Create the Form to Add a Clinic
## Value Statement 

As a user 
I want to be able to add a clinic
so I have an overview of all the clinics I get information once. 

## Description

image

## Acceptance Critera

- create a Form 
- link it to the other pages
- styled it with css 

## Tasks

- [x] create a new page
- [x] create a Add-Card Component

### PR DESCRIPTION
**Pull request by Shari-alt**
Finally add the clinic form to add a clinic. 
I implemented use history and did a lot of css. 
In the next step the local storage should be implemented to actually add a clinic. 

Thank you guys for you review 🌻 & have a nice weekend 🍻

### CODE REVIEW

File: src/components/Add.css [hunk 1]
```diff
@@ -0,0 +1,253 @@
...
+.InsuranceSelect {
+  border: none;
+  border-radius: 15px;
+  background-color: #e2ecf8;
+  box-shadow: 8px 8px 8px #babdc0, inset -8px -8px 8px #e2ecf8;
+  margin-top: 18px;
+  padding: 15px;
+  width: 267px;
+  height: 90px;
+  justify-content: space-between;
```
Comment (FriederikeM) [inline src/components/Add.css:69]: Not too sure you can use this, if .InsuranceSelect isn't a flex/grid. I might be wrong though

File: src/components/Add.css [hunk 2]
```diff
@@ -0,0 +1,253 @@
...
+
+.FormTitle {
+  font-size: 1.3em;
+  margin-bottom: 8px;
+  text-align: center;
+}
+
+.TherapyCheckbox {
+  display: grid;
+  grid-template-columns: [firstCol] 80px [centerCol] 80px [leftCol] 80px;
```
Comment (FriederikeM) [inline src/components/Add.css]: here you say firstCol, but below you say LeftCol, try keeping it consistent

File: src/components/Add.css [hunk 3]
```diff
@@ -0,0 +1,253 @@
...
+.Room {
+  border: none;
+  border-radius: 15px;
+  background-color: #e2ecf8;
+  box-shadow: 8px 8px 8px #babdc0, inset -8px -8px 8px #e2ecf8;
+  margin-top: 18px;
+  padding: 4px 15px 4px 15px;
+  width: 170px;
+  height: 30px;
+}
```
Comment (FriederikeM) [inline src/components/Add.css]: this looks all VERY similiar. you could save a lot of lines if you say
```
.Visitors,
.Children,
.Animals,
.Room {
border: none;
border-radius: 15px
etc ... 
} 
```
and then, for the properties they differ have their own selector:
```
.Children {
padding: 2px 10px 2px 10px;
}
```

**Review by FriederikeM — APPROVED** [review]
This looks really good Shari, good job on the form and the styling. I just left a little comment t0 save some lines :)

File: src/components/Add.css (hunk 3 above)
Comment (Shari-alt) [reply src/components/Add.css]: Thank you Rikki!

**Pull request merged**